docs/source/daml/intro/5_Restrictions.rst

```diff
@@ -66,34 +66,38 @@ For example, the simple Iou in :ref:`simple_iou` allowed the no-op where the ``o
+Similarly, you can write a ``Redeem`` choice, which allows the ``owner`` to redeem an ``Iou`` *during business hours on weekdays*. Notice the ``Redeem`` choice implementation below confirms that ``getTime`` returns a value that is during business hours on weekdays. If all those checks pass, it doesn't do anything other than archive the ``SimpleIou``. (This assumes that actual cash changes hands off-ledger:)
 
 .. literalinclude:: daml/daml-intro-5/daml/Restrictions.daml
   :language: daml
   :start-after: -- REDEEM_CHOICE_BEGIN
   :end-before: -- REDEEM_CHOICE_END
 
+There's quite a lot going on inside the ``do`` block of the ``Redeem`` choice, with several uses of the ``<-`` operator. ``do`` blocks and ``<-`` deserve a proper explanation which we will get to below. For now, let's look at how the ``Redeem`` choice is exercised in a script.
+
 .. literalinclude:: daml/daml-intro-5/daml/Restrictions.daml
   :language: daml
   :start-after: -- REDEEM_TEST_BEGIN
   :end-before: -- REDEEM_TEST_END
 
-There are quite a few new time-related functions from the ``DA.Time`` and ``DA.Date`` libraries here. Their names should be reasonably descriptive so how they work won't be covered here, but given that Daml assumes it is run in a distributed setting, we will still discuss time in Daml.
+In the above example, the time is taken apart into day of week and hour of day using standard library functions from ``DA.Date`` and ``DA.Time``. The hour of the day is checked to be in the range from 8 to 18.
 
-There's also quite a lot going on inside the ``do`` block of the ``Redeem`` choice, with several uses of the ``<-`` operator. ``do`` blocks and ``<-`` deserve a proper explanation at this point.
+There are quite a few new time-related functions from the ``DA.Time`` and ``DA.Date`` libraries here. Their names should be reasonably descriptive so how they work won't be covered here, but given that Daml assumes it is run in a distributed setting, we will still discuss time in Daml.
 
 Time on Daml Ledgers
 --------------------
 
-Each transaction on a Daml ledger has two timestamps called the *ledger time (LT)* and the *record time (RT)*. The ledger time is set by the participant, the record time is set by the ledger.
+Each transaction on a Daml ledger has two timestamps. One of the timestamps is called the *ledger time (LT)*. The other is called the *record time (RT)*.
+
+**Ledger time (LT)** is the time associated with a transaction in the abstract ledger model. It is the time of a transaction from a business and application perspective. The LT time is determined by the participant. When you call ``getTime``, it is the LT time that is returned. The LT is used when reasoning about related transactions and commits. The LTs can be compared with other LTs to guarantee model consistency. For example, LTs are used to enforce that no transaction depends on a contract that does not exist. This is the requirement known as "causal monotonicity."
```

Comment:
I would drop the word "abstract" from the phrase "abstract ledger model". The word connotes "less real", when in fact for the purposes of Daml the LT is _more_ real. 

```suggestion
**Ledger time (LT)** is the time associated with a transaction in the ledger model, as determined by the participant. It is the time of a transaction from a business and application perspective. When you call ``getTime``, it is the LT time that is returned. The LT is used when reasoning about related transactions and commits and is compared with other LTs to guarantee model consistency. For example, LTs are used to enforce that no transaction depends on a contract that does not exist. This is the requirement known as "causal monotonicity."
```

Reply:
Fixed with 2e4aa2da89cdde635c9b729ef5f7f13f0adf0a22